### ISSUE TITLE
Clarify ChanServ usage and Flags

### ISSUE THREAD

**Issue by 0x5c** [Libera-Chat/libera-chat.github.io#60]
The Channel Modes guide mentions ChanServ flags, but not what they mean and how to use them. There is also no info anywhere in the guides about using ChanServ beyond registering a channel.

**Comment by jmbwell** [comment]
made an attempt at this. submitted in the hope it is useful!

**Comment by Mikaela** [comment]
Would this document also include how to find who are ops on a channel? Ref: https://github.com/Libera-Chat/libera-chat.github.io/pull/135/files#r692104401

**Comment by emersonveenstra** [comment]
> Would this document also include how to find who are ops on a channel? Ref: https://github.com/Libera-Chat/libera-chat.github.io/pull/135/files#r692104401

No, this is for people configuring their channel. Finding ops is covered here: https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted

**Comment by meldra** [comment]
The current https://libera.chat/guides/creatingchannels was written quickly and should be rewritten, it'd ideally include the above information.
